{
  "Io-language": null,
  "clang": null,
  "git-filter-repo": "https://bugzilla.redhat.com/show_bug.cgi?id=2251901",
  "gnome-builder": null,
  "gpodder": null,
  "libguestfs": "https://bugzilla.redhat.com/show_bug.cgi?id=2256945",
  "libiio": null,
  "libnuml": null,
  "libucl": "https://bugzilla.redhat.com/show_bug.cgi?id=2251905",
  "mapserver": null,
  "mat2": "https://bugzilla.redhat.com/show_bug.cgi?id=2252077",
  "nut": "https://bugzilla.redhat.com/show_bug.cgi?id=2248125",
  "policycoreutils": null,
  "pyserial": "https://bugzilla.redhat.com/show_bug.cgi?id=2245859",
  "python-asyncssh": "https://bugzilla.redhat.com/show_bug.cgi?id=2251916",
  "python-automaton": null,
  "python-boto3": null,
  "python-cachelib": "https://bugzilla.redhat.com/show_bug.cgi?id=2251780",
  "python-can": "https://bugzilla.redhat.com/show_bug.cgi?id=2255345",
  "python-cmake-build-extension": null,
  "python-dirty-equals": null,
  "python-fiona": null,
  "python-frozenlist": null,
  "python-google-cloud-bigquery-datatransfer": null,
  "python-google-cloud-bigquery-reservation": null,
  "python-google-cloud-billing": null,
  "python-google-cloud-build": null,
  "python-google-cloud-firestore": "https://bugzilla.redhat.com/show_bug.cgi?id=2254981",
  "python-google-cloud-org-policy": null,
  "python-heapdict": "https://bugzilla.redhat.com/show_bug.cgi?id=2251790",
  "python-html5-parser": null,
  "python-ironicclient": null,
  "python-jupyterlab_pygments": null,
  "python-lazy-object-proxy": null,
  "python-oauthlib": "https://bugzilla.redhat.com/show_bug.cgi?id=2247024",
  "python-oslo-i18n": null,
  "python-param": null,
  "python-pydantic": "https://bugzilla.redhat.com/show_bug.cgi?id=2256753",
  "python-pynamodb": null,
  "python-pytest-mock": "https://bugzilla.redhat.com/show_bug.cgi?id=2251908",
  "python-pytz-deprecation-shim": null,
  "python-rstcheck-core": null,
  "python-sphinx-hoverxref": null,
  "python-testtools": null,
  "python-wrapt": "https://bugzilla.redhat.com/show_bug.cgi?id=2256756",
  "python-zaqarclient": null,
  "python-zbase32": null,
  "python-zope-interface": "https://bugzilla.redhat.com/show_bug.cgi?id=2246286",
  "python3.13": null,
  "recoll": "https://bugzilla.redhat.com/show_bug.cgi?id=2245810",
  "rmol": null,
  "scipy": null,
  "sigil": null,
  "texworks": null
}